workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/Generation/Model/Details/DetailsStore.cs

```diff
@@ -105,7 +105,11 @@ private void PopulateBlittableMaps()
 
             foreach (var component in bundle.BundleContents.ComponentDefinitions)
             {
-                var blittableCheckResult = CheckBlittable(component.Fields);
+                var fields = component.Data != null
+                    ? bundle.BundleContents.TypeDefinitions.First(type => type.Identifier.QualifiedName == component.Data.QualifiedName).Fields
```

Comment:
Can `First` have no result? If so, will this throw?

Reply:
If it has no result the bundle is invalid. 

We could probably handle this more gracefully however.